plugins/snowflake/dbt/include/snowflake/macros/materializations/incremental.sql

```diff
@@ -37,6 +37,7 @@
 
   {#-- Validate early so we don't run SQL if the strategy is invalid --#}
   {% set strategy = dbt_snowflake_validate_get_incremental_strategy(config) -%}
+  {% set on_schema_change = incremental_validate_on_schema_change(config.get('on_schema_change')) %}
```

Comment:
If we choose to roll back the addition of `on_schema_change` as a NodeConfig, we'll need to set the default here instead:
```suggestion
  {% set on_schema_change = incremental_validate_on_schema_change(config.get('on_schema_change'), default = 'ignore') %}
```